crates/matrix-sdk/src/widget_api/api/mod.rs

```diff
@@ -132,7 +132,7 @@ async fn process_responses(
 async fn forward(mut receiver: Receiver<impl Into<WidgetMessage>>, sink: Sender<String>) {
     while let Some(msg) = receiver.recv().await {
         let raw = to_json(&msg.into()).expect("Bug: invalid JSON");
-        let _ = sink.send(raw);
+        let _ = sink.send(raw).map_err(|e|println!("could not forward message: {}", e));
```

Comment:
We should not use `println!()` in a library code, not the app code (the app may not want that a library writes to the stdout that may be used by other code that e.g. writes formatted output to the stdout, also, there might be no stdout to begin with).